bigquery_etl/cli/query.py

```diff
@@ -704,9 +704,9 @@ def deploy(ctx, name, sql_dir, project_id):
 
                 if metadata.bigquery and metadata.bigquery.time_partitioning:
                     new_table.time_partitioning = bigquery.TimePartitioning(
-                        metadata.bigquery.time_partitioning.type.bigquery_type(),
```

Comment:
This was raising an error. `bigquery_type` is defined with a `@property` annotation.